README.md

```diff
@@ -0,0 +1,35 @@
+# integration test runner bot
+
+## Infrastructure
+It's currently hosted on `company-websites` GKE Kubernetes cluster. The cluster in still in POC phase, so, CD is simplified to not have a lot of time vasted if the POC won't go live.
+
+## Continuous Delivery
+
+### Setup access to GKE
+1. create service account with the following roles assigned: `Kubernetes Engine Developer`, `Kubernetes Engine Service Agent` and `Viewer`
+2. create json key and make base64 encoded hash with removing new lines: `base64 /path/to/saved-key.json | tr -d \\n`
+3. in CI/CD project settings add `GCLOUD_SERVICE_KEY` variable where value is the hash
```

Comment:
Okey for now, to be moved to a more generic place after the PoC

Reply:
Good point. Also thought about it 👌